cli/dts/lib.deno.ns.d.ts

```diff
@@ -3394,18 +3394,19 @@ declare namespace Deno {
     /** Clean up resources associated with the sub-process instance. */
     close(): void;
     /** Send a signal to process.
+     * Default signal is `"SIGTERM"`.
```

Comment:
Maybe the intellisense/generated documentation is better if we do something like the following?

```ts
* @param [signo="SIGTERM"] - Signal to send to the process
```

https://jsdoc.app/tags-param.html

Reply:
oh interesting, didnt notice thats a thing. quickly checked, and deno_doc doesn't support it, so going to look into adding that first

Reply:
Maybe we should wait for https://github.com/microsoft/tsdoc/issues/151 actually. Just noticed that.

Reply:
hm, looking at that issue, made me notice microsoft/tsdoc#27, which we actually recently implemented... so maybe it would be fine? because i am in the need for default value for parameters for various documentation related things, so ideally we would implement it as per jsdoc spec. because i honestly have little hope that those tsdoc issues will get resolved soon and being blocked on those for a massive improvement in documentation would just be silly to me...